testing/README.md

```diff
@@ -0,0 +1,37 @@
+# Git-Brunching Integrated Test Automation
+
+## Table of Content
+
+* [General Information](#general-info)
+* [Technologies And Version](#tech-and-version)
+* [Setup](#setup)
+* [License](#license)
+
+## General info
+
+This is an automated integration test suite for git-brunching project. The purpose of this suite is to perform integrated tests.
+
+## Tech and version
+
+* NodeJS - 12.14.0
+* Cypress - 4.1.0
+
+## Stepup
+
+* NodeJS:
+
+Download the installer from the [official Node.js website](https://nodejs.org/en/) and follow the installation instructions. This will also install npm.
+
+* Cypress:
+
+Install Cypress via `npm`
```

Comment:
Good work Tim. I was just wondering if this is necessary? Can't we just do `npm i`  which should take care of all dependencies?

Reply:
Yes, you are right, a npm install will do the work. I will modify this.